dolphin/api/v1/alert.py

```diff
@@ -127,13 +147,17 @@ def _input_check(self, alert_source):
 
         return alert_source
 
-    def _exist(self, ctx, storage_id):
+    def _get_snmp_config_brief(self, ctx, storage_id):
         try:
-            db.alert_source_get(ctx, storage_id)
+            alert_source = db.alert_source_get(ctx, storage_id)
+            snmp_config = {"storage_id": alert_source["storage_id"],
+                           "version": alert_source["version"]}
+            if snmp_config["version"].lower() == "snmpv3":
```

Comment:
How about v2 case here, community name to be filled?

Reply:
For Snmp v1 and v2, only community_index is required to delete from trap receiver, as we used storage_id as the community_index, so no need to fill commmunity string here.